packages/api-nest/src/library/library.resolver.ts

```
@@ -68,6 +77,72 @@ export class LibraryResolver {
+  @Query(() => SearchResult, {
+    name: 'search',
+    description: 'Legacy search query for backward compatibility',
+  })
+  @UseGuards(JwtAuthGuard)
+  async search(
+    @CurrentUser() user: User,
+    @Args('first', { type: () => Int, nullable: true, defaultValue: 20 })
+    first = 20,
+    @Args('after', { type: () => String, nullable: true }) after?: string,
+    @Args('query', { type: () => String, nullable: true }) query?: string,
+    @Args('includeContent', { type: () => Boolean, nullable: true, defaultValue: false })
```

ESLint

[error] Replace `·type:·()·=>·Boolean,·nullable:·true,·defaultValue:·false` with `⏎······type:·()·=>·Boolean,⏎······nullable:·true,⏎······defaultValue:·false,⏎···`

(prettier/prettier)